### DISCUSSION TITLE
Vue 3 support

### DISCUSSION THREAD

**Discussion by LucasFlaquer** [timtnleeProject/vuejs-clipper#83] [General]
Is this package compatible with the vue in his newest version?

Is it  compatible with typescript too?

**Comment by timtnleeProject** [comment]
Hi @LucasFlaquer ,
sorry but for now, it's not support vue3 and typescript :(

**Comment by LucasFlaquer** [comment]
Hi, thanks for the answer.

Could you please let me know when those features will be implemented? 
I would like to test it as soon as it is implemented

**Comment by timtnleeProject** [comment]
@LucasFlaquer 
These would be great features,
unfortunately I have no time and plan for them now...

And I'm not sure whether to make a vue3 migration.
I'm going to suspend this issue for more time to think and more feedback, thanks!

**Comment by mmgolden** [comment]
@timtnleeProject Any update on the compatibility with Vue 3?
